pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2654,6 +2654,36 @@ func updatePortAndPciBackIoBundleAll(ctx *domainContext) {
 	}
 }
 
+// nvmeDeviceIsMountedByEVE checks if an NVME device is used by EVE-OS
+func nvmeDeviceIsMountedByEVE(pciID string) bool {
+	// Find /dev/nvmX from pciID
+	// e.g., ls /sys/bus/pci/devices/<pciID>/nvme/
+	//  -> nvme1
+	cmd := "/bin/ls"
```

Comment:
No need to exec a binary to read a file or symlink. The os and io packages have all that is needed to do this.

Reply:
Note that we already have similar code in ifnametopci.go such as PCIGetIOMMUGroup()
Makes sense following the same pattern and putting it in that file.